x-pack/plugins/security/server/lib/authentication/session.ts

```diff
@@ -4,13 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { Request } from 'hapi';
 import hapiAuthCookie from 'hapi-auth-cookie';
+import { Legacy } from 'kibana';
 
 const HAPI_STRATEGY_NAME = 'security-cookie';
 // Forbid applying of Hapi authentication strategies to routes automatically.
 const HAPI_STRATEGY_MODE = false;
 
-function assertRequest(request) {
+/**
+ * The shape of the session that is actually stored into cookie.
+ */
+interface InternalSession {
+  /**
+   * Session value that is fed to the authentication provider. The shape is unknown upfront and
+   * entirely determined by the authentication provider that owns current session.
+   */
+  value: unknown;
+
+  /**
+   * The date in ms when session should be considered expired. If `null` session will stay active
```

Comment:
```suggestion
   * The duration in ms when the session should be considered expired. If `null`, session will stay active
```

Reply:
Actually it's not  `date in ms` or `duration in ms`, it's `Unix time in ms`: `expires: this.ttl && Date.now() + this.ttl` :) Will use that term.